scripts/ingests/Gaia/ingest_gaiadr3.py

```diff
@@ -0,0 +1,89 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
```

Comment:
May be good to explicitly list which functions we are using from here rather than importing *